.github/workflows/build.yml

```diff
@@ -0,0 +1,69 @@
+name: build
+
+on: push
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Node 12.7
```

Comment:
For the Lint job, let's run it in the current LTS (long term support) version of Node, which is `14`.